partcad-service-json-rpc/src/partcad_service_json_rpc/rpc/dispatcher.py

```
@@ -0,0 +1,88 @@
+            method = request["method"]
+            handler = self._registry.get(method)
+            if handler is None:
+                raise JsonRpcError(METHOD_NOT_FOUND, f"Method not found: {method}")
+
+            params = request.get("params", {})
+            result = handler(session, params)
+        except JsonRpcError as e:
+            if is_notification:
+                return None
+            return {"jsonrpc": "2.0", "id": request_id, "error": e.to_object()}
+        except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)